frontend/src/metabase/search/components/filters/TypeFilter/TypeFilter.tsx

```diff
@@ -22,7 +22,7 @@ export const TypeFilter: SearchFilterComponent<"type"> = {
     }
     return isEnabledSearchModelType(value) ? [value] : [];
   },
-  toUrl: (value?: TypeFilterProps) => {
```

Comment:
I wanted to remove the use of `undefined` since it's bad practice to actually declare a value as `undefined`, so I switching things to nulls